.Gruntfile.babel.js

```diff
@@ -253,10 +253,16 @@ module.exports = (grunt) => {
     const cypress = require('cypress')
     const done = this.async()
     const command = grunt.option('browser') === 'debug' ? 'open' : 'run'
+    const disableVideo = process.argv.indexOf('--disable-video') > -1
+
+    console.log('disableVideo', disableVideo, process.argv)
```

Comment:
No debug logging should be here. And if you absolutely require it, it should use the appropriate `grunt.log.debug` type stuff so that it's not visible by default.